src/common/types/store-data/unified-data-interface.ts

```diff
@@ -91,19 +92,17 @@ export type UnifiedDescriptors = {
     boundingRectangle?: BoundingRectangle;
 } & InstancePropertyBag;
 
+export type UnifiedRichResolution = {
+    type: 'check' | 'fix';
```

Comment:
Calling this `type` made me think that there is some sort of logical significance to the value, but it looks like it's just used for the How to __ row label. Do you think a name like `labelType` (or something else that suggests this is just a UI string) might be better? I'm happy either way